l10n_ch_account_statement_base_import/account_bank_statement_import.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, api, _, exceptions, fields
+from .parser import base_parser
+
+
+class AccountBankStatement(models.Model):
+    _inherit = 'account.bank.statement'
+
+    related_files = fields.Many2many(
+        comodel_name='ir.attachment',
+        string='Related files',
+        readonly=True
+    )
+
+
+class AccountBankStatementLine(models.Model):
+    _inherit = 'account.bank.statement.line'
+
+    related_file = fields.Many2many(
```

Comment:
Can a line have several attachments ? If yes, maybe we could rename this field to `related_files`.
If not, maybe we could change to `Many2one` ?